crates/accelerate/src/two_qubit_decompose.rs

```diff
@@ -182,9 +182,25 @@ fn decompose_two_qubit_product_gate(
     }
     l.mapv_inplace(|x| x / det_l.sqrt());
     let phase = det_l.arg() / 2.;
+
     Ok((l, r, phase))
 }
 
+#[pyfunction]
+#[pyo3(signature = (special_unitary))]
+fn py_decompose_two_qubit_product_gate(
+    py: Python,
+    special_unitary: PyArrayLike2<Complex64>,
+) -> PyResult<(PyObject, PyObject, f64)> {
+    let view = special_unitary.as_array();
+    let (l, r, phase) = decompose_two_qubit_product_gate(view)?;
+    Ok((
+        PyObject::from(l.into_pyarray_bound(py).unbind()),
+        PyObject::from(r.into_pyarray_bound(py).unbind()),
```

Comment:
This is fine, but you can also probably just do:

```suggestion
        l.into_pyarray_bound(py).unbind().into(),
        r.into_pyarray_bound(py).unbind().into(),
```